sympy/crypto/crypto.py

```diff
@@ -3324,3 +3324,130 @@ def decipher_bg(message, key):
         orig_msg += (m ^ b)
 
     return orig_msg
+
+def create_matrix(key):
+    #helper function for enciphering and deciphering playfair cipher
+
+    alphabet = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
+    temp = ''
+
+    key = key.lower()
+
+    for i in range(0, len(key)):
+        if key[i] not in temp:
+            temp += key[i]
+
+    temp.replace(" ","")
+
+    for i in alphabet:
+        if i not in temp and i != 'j':
+            temp += i
+
+    matrix = [[0]*5 for i in range(5)]
+
+    for i in range(0, len(temp)):
+        row = int(i / 5)
+        col = i % 5
+        matrix[row][col] = temp[i]
+
+    return matrix
+
+
+def group_letters(text):
+    #helper function for enciphering and deciphering playfair cipher
+
+    text = text.lower()
+
+    if len(text)%2 != 0:
+        text += 'z'
+
+    groups = []
+
+    for i in range(0, int(len(text)/2)):
+        groups.append(text[2*i] + text[2*i + 1])
+
+    return groups
+
+def encipher_playfair(key, plaintext):
+    #encrypt text
+
+    key_matrix = create_matrix(key)
+    text = group_letters(plaintext)
+
+    ciphertext = []
+
+    for i in text:
+        for j in range(0, 5):
+            try:
+                fcoords = ( j, key_matrix[j].index(i[0]) )
+                if fcoords[0] != -1:
+                    break
+            except ValueError:
+                continue
```

Comment:
Can we use `if else` instead of `try except`? Its slow and masks other bugs as well.